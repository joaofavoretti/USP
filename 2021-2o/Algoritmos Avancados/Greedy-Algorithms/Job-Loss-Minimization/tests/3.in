3
100
4 76
22 58
11 82
2 67
14 75
22 56
10 35
43 82
13 32
30 83
49 94
43 35
16 18
32 62
22 86
20 27
13 44
37 26
26 39
43 92
14 17
1 26
2 96
9 16
29 41
1 78
37 94
15 3
14 98
17 87
34 37
16 99
33 3
25 59
43 70
4 9
38 5
36 40
1 97
8 82
40 59
13 77
6 78
33 22
26 2
9 13
42 75
14 25
31 42
37 74
12 41
35 52
48 23
43 51
20 53
34 62
15 47
42 23
25 25
45 54
27 7
19 19
34 83
45 65
27 34
39 90
25 75
44 25
48 39
26 71
43 10
35 10
9 77
33 87
2 30
41 82
37 12
1 72
47 98
37 25
32 29
17 59
6 61
34 6
1 61
29 94
23 15
6 33
42 89
22 95
21 13
29 9
27 30
33 24
31 22
49 65
3 16
27 59
28 61
18 79
15
16 71
33 78
4 60
38 8
43 19
6 89
44 51
38 22
29 51
47 43
43 49
49 61
32 39
36 55
6 66
80
44 4
6 65
45 60
22 26
20 64
43 18
10 45
28 41
34 4
23 45
48 41
12 8
39 83
9 50
48 24
18 94
28 74
9 73
36 83
2 57
49 95
27 59
41 7
1 77
12 76
22 10
19 85
20 8
18 31
11 18
5 31
13 36
7 24
9 44
10 61
3 59
9 32
21 50
40 23
29 4
49 2
14 21
39 37
31 7
18 92
27 76
23 42
12 81
19 73
27 79
37 82
38 96
14 9
48 6
33 77
10 84
31 77
7 70
16 39
29 86
31 7
12 7
49 26
38 68
1 65
49 87
49 38
35 13
49 33
22 84
12 34
21 94
13 78
14 29
17 94
18 1
3 30
8 2
9 48
23 61
