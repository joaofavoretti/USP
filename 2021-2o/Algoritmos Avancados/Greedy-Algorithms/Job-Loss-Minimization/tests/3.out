3361785
103294
1750469
